pytest/tests/shardnet/restake.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import shlex
+import random
+import sys
+from rc import pmap
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+import mocknet
+from configured_logger import logger
+
+
+def restart_restaked_script(restaked_url, delay_sec):
+    s = '''
+        killall restaked
+        wget '{restaked_url}' -O /home/ubuntu/restaked
+        nohup ./restaked --home /home/ubuntu/.near/shardnet/ --rpc-url 127.0.0.1:3030 --wait-period {delay_sec} --stake-amount {stake_amount} 1>>/home/ubuntu/restaked.out 2>>/home/ubuntu/restaked.err </dev/null &
+    '''.format(restaked_url=shlex.quote(restaked_url),
+               stake_amount=shlex.quote(str(random.randint(10**3, 10**5))),
+               delay_sec=shlex.quote(str(delay_sec)))
+    logger.info(f'Starting restaked: {s}')
+    return s
+
+
+def create_account(node, near_pk, near_sk):
```

Comment:
I'm surprised that this executes the code on the remote machine.. please add comments (either here or on the top of file)